### PR TITLE
New blog post: Node-RED for PLC

### PR DESCRIPTION
**Pull request by iskerrett**
## Description

New blog post talking about using Node-RED as an integration platform for edge computing and PLCS.

## Checklist

 - [X ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2023/03/integration-platform-for-edge-computing.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: 'Node-RED: The Integration Platform for IIoT Edge Computing & PLCs'
+subtitle: 
+description: News from the FlowForge and Node-RED communities
+date: 2023-03-06
+authors: ["ian-skerrett"]
+image: /blog/2023/03/images/integration-platform-blog-image.jpg
+---
+
+Node-RED has become a widely adopted integration platform for IoT edge computing and PLCs. Discover why?
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/integration-platform-for-edge-computing.md:10]:
```suggestion
Node-RED has become a widely adopted integration platform for IoT edge computing and PLCs. Discover why!
```

File: src/blog/2023/03/integration-platform-for-edge-computing.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+<!--more-->
+
+
+## The Integration Platform for IIoT Edge Compuuting & PLCs
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/integration-platform-for-edge-computing.md:15]:
```suggestion
## The Integration Platform for IIoT Edge Computing & PLCs
```

File: src/blog/2023/03/integration-platform-for-edge-computing.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+14. [Revolution Pi](https://revolutionpi.com/revpi-connect/) RevPi Connect
+15. [Siemens](https://github.com/SIMATICmeetsLinux/IOT2050-NodeRed-OPCUA-Server) S7 PLC
+16. [Tulip](https://support.tulip.co/docs/using-node-red-with-edge-mc) Edge MC
+17. [Wago](https://www.wago.com/us/edge-devices) Edge Controller & Computer
+18. [Weidmueller](https://catalog.weidmueller.com/procat/Group.jsp;jsessionid=C885C404E7B4B798B23B8A9BB2200513?groupId=(%22group14048963834797%22)&page=Group) control web
+
+There are several key reason Node-RED is so popular for IIoT edge computing, including:
+
+
+
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/integration-platform-for-edge-computing.md:66]:
```suggestion
```

File: src/blog/2023/03/integration-platform-for-edge-computing.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+## The Standard for Edge Computing and PLCs
+
+Today, Node-RED has been adopted by some of the leading PLC and IoT Gateways vendors. The hardware vendor community appears to have standardized on Node-RED as being the edge computing platform for IIoT.
+
+Below is a sample of the vendors offering a Node-RED solution:
+
+
+
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/integration-platform-for-edge-computing.md:43]:
```suggestion

```

File: src/blog/2023/03/integration-platform-for-edge-computing.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Node-RED has become a widely adopted integration platform for IoT edge computing and PLCs. Discover why!
+
+<!--more-->
+
+
+## The Integration Platform for IIoT Edge Computing & PLCs
+
+Node-RED is a widely adopted open-source low-code development tool that makes it easy to connect and integrate different sources of data. With a visual programming interface and drag-and-drop functionality, [Node-RED](https://flowforge.com/node-red/) makes it possible for software developers and non-professional software developers to create sophisticated applications. 
+
+In the manufacturing and industrial automation industry, the focus of the Industrial Internet of Things (IIoT) has been on integrating industrial processes and equipment to enable real-time monitoring, control, and analysis of data. For many use cases, instead of sending all the data to the cloud, the best practice for processing industrial data is to deploy the application to the edge of the network, referred to as edge computing. By processing the data closer to the data source, edge computing has many benefits, including reduced latency, limited downtown, conserving bandwidth, and increased privacy, and security.
```
Comment (robmarcer) [inline src/blog/2023/03/integration-platform-for-edge-computing.md:19]: I assume this should say limited downtime not downtown?

File: src/blog/2023/03/integration-platform-for-edge-computing.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+## PLC and IoT Gateway Vendors Embrace Node-RED
+
+PLC and IoT Gateway vendors are at the forefront of promoting edge computing. They see edge computing as a way to modernize hardware in the factory and for remote asset management. PLC and IoT gateways often sit in front of old legacy systems that don’t have the connectivity or compute platform to enable IIoT applications. 
+
+Many of these hardware vendors realized they need an application delivery platform for their devices. Traditional OT hardware vendors often implement proprietary software stacks that are often difficult to use and closed to integrating with other hardware and software. Forward thinking hardware vendors realized having an open platform is the future of their industry and customers have begun to demand more open platforms. Node-RED’s ease of use, open community and open source license provided the solution many of these hardware vendors were looking for. 
+
+
+## The Standard for Edge Computing and PLCs
+
+Today, Node-RED has been adopted by some of the leading PLC and IoT Gateways vendors. The hardware vendor community appears to have standardized on Node-RED as being the edge computing platform for IIoT.
```
Comment (robmarcer) [inline src/blog/2023/03/integration-platform-for-edge-computing.md:38]: Should be Gateway rather than Gateways I think.

**Review by robmarcer — CHANGES_REQUESTED** [review]

File: src/blog/2023/03/integration-platform-for-edge-computing.md (hunk 5 above)
Comment (iskerrett) [reply src/blog/2023/03/integration-platform-for-edge-computing.md:19]: fixed

File: src/blog/2023/03/integration-platform-for-edge-computing.md (hunk 6 above)
Comment (iskerrett) [reply src/blog/2023/03/integration-platform-for-edge-computing.md:38]: fixed

**Review by robmarcer — APPROVED** [review]

**Pull request merged**